### PR TITLE
Update metrics-definitions.md

### PR DESCRIPTION
**Pull request by wiersgallak**
#### Summary
Updated MAU based on this conversation https://community-daily.mattermost.com/private-core/pl/jrjwd76mobne7e6baxtsmg1b8r of how reactivated and deactivating users in Mattermost affects the MAU statistic.

### CODE REVIEW

**Review by michaelschiffmm — DISMISSED** [review]

File: operations/business-operations/analytics/metrics-definitions.md
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query. However, reactivating a user in Mattermost will increase the MAU statistic. 
```

**Review by justinegeffen — DISMISSED** [review]

**Comment by jasonblais** [comment]
@wiersgallak is this one ready to merge, or should we get Justine and Schiff do a second review?

**Comment by wiersgallak** [comment]
I think @enelson720 still needs to review.  Here were some notes he provided me when we chatted via MM: 

> I think you have it added to the incorrect section. MAU is the user event-based MAU metric. active_users_montly,  active_users_daily, and active_user_count measures are from security and activity diagnostic telemetry and those are the measures impacted by deactivation. I also acknowledge that there are way too many variations of active users

File: operations/business-operations/analytics/metrics-definitions.md
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to Rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
```

File: operations/business-operations/analytics/metrics-definitions.md
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
* Number of trial license requests via [https://mattermost.com/trial](https://mattermost.com/trial) from Named Accounts or Enterprises with 5,000+ employees, in America, EMEA, Australia, or Japan.
```

**Comment by justinegeffen** [comment]
@enelson720, following up on this to confirm whether it's ready to merge.

File: operations/business-operations/analytics/metrics-definitions.md
Comment (justinegeffen) [inline operations/business-operations/analytics/metrics-definitions.md]:
```suggestion
Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query. However, reactivating a user in Mattermost will increase the MAU statistic.
```

**Review by justinegeffen — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Gentle reminder, @enelson720 and @michaelschiffmm. :)

**Comment by enelson720** [comment]
> Gentle reminder, @enelson720 and @michaelschiffmm. :)

Sorry, just saw this. Reviewing now.

**Review by enelson720 — CHANGES_REQUESTED** [review]
This portion of MAU is for user-level MAU events. The server-level diagnostics send `active_users_monthly` statistics that are impacted by the activating and deactivating of users, but this should not impact the user-level MAU calculation, so I don't believe this would be the correct place to have the callout. Perhaps moving it to the TEDAU section as the process of activating and deactivating users also impacts it as well. @justinegeffen

**Review by michaelschiffmm — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Thanks, @enelson720. I've moved the section as suggested.

One other note is that in some cases the dollar value is presented as "30k" and in others it's "$30k" (as an example). If you are able to review that and confirm whether that is correct or they should all have a $ sign that'd be great. :)

**Review by michaelschiffmm — DISMISSED** [review]

**Comment by justinegeffen** [comment]
hey @enelson720. Just following up on this to find out whether we can merge or should close this PR. :)

**Review by enelson720 — APPROVED** [review]
LGTM

**Pull request merged**
